generators/client-2/templates/src/main/webapp/app/components/_common.module.ts

```diff
@@ -36,7 +36,7 @@ angular
     .factory('LoginService', LoginService)
     .factory('Principal', Principal)
     .factory('ProfileService',upgradeAdapter.downgradeNg2Provider(ProfileService))
-    .provider('AlertService', AlertService)<% if (enableTranslation) { %>
+    .provider('AlertService', AlertService) <% if (enableTranslation) { %>
     .factory('<%=jhiPrefixCapitalized%>LanguageService', <%=jhiPrefixCapitalized%>LanguageService)<% } %>
-    .directive('pageRibbon', upgradeAdapter.downgradeNg2Component(PageRibbonComponent))<% if (enableTranslation) { %>
-    .controller('<%=jhiPrefixCapitalized%>LanguageController', <%=jhiPrefixCapitalized%>LanguageController);<% } %>
+    .directive('pageRibbon',  <angular.IDirectiveFactory> upgradeAdapter.downgradeNg2Component(PageRibbonComponent))<% if (enableTranslation) { %>
+    .controller('<%=jhiPrefixCapitalized%>LanguageController', <%=jhiPrefixCapitalized%>LanguageController)<% } %> ;
```

Comment:
@sendilkumarn not being too picky but couldnt help noticing it, there is a stray space at the end before the semicolon :smiling_imp: